internal/querycoord/task_scheduler.go

```diff
@@ -924,7 +924,8 @@ func updateSegmentInfoFromTask(ctx context.Context, triggerTask task, meta Meta)
 		rollBackSegmentChangeInfoErr := retry.Do(ctx, func() error {
 			rollBackChangeInfos := reverseSealedSegmentChangeInfo(sealedSegmentChangeInfos)
 			for collectionID, infos := range rollBackChangeInfos {
-				_, _, sendErr := meta.sendSealedSegmentChangeInfos(collectionID, infos)
+				channelInfo := meta.getQueryChannelInfoByID(collectionID)
```

Comment:
check the error handle logic there, no matter if error happens it print "Successfully roll back segment info change"
		if rollBackSegmentChangeInfoErr != nil {
			log.Error("scheduleLoop: Restore the information of global sealed segments in query node failed", zap.Error(rollBackSegmentChangeInfoErr))
		}
		log.Info("Successfully roll back segment info change")
		return err

Reply:
get rollBackSegmentChangeInfoErr after retry, so should check error after retry done. I have change the error check to 
```
		if rollBackSegmentChangeInfoErr != nil {
			log.Error("scheduleLoop: Restore the information of global sealed segments in query node failed", zap.Error(rollBackSegmentChangeInfoErr))
			panic(rollBackSegmentChangeInfoErr)
		} else {
			log.Info("Successfully roll back segment info change")
		}
```